app/scripts/modules/core/src/plugins/plugin.registry.spec.ts

```diff
@@ -5,18 +5,14 @@ import { IPluginMetaData, PluginRegistry } from './plugin.registry';
 import { Registry } from 'core/registry';
 import { mock } from 'angular';
 
-type TestPluginRegistry = PluginRegistry & {
-  load(pluginMetaData: IPluginMetaData): Promise<any>;
-  loadModuleFromUrl(url: string): Promise<any>;
-};
```

Comment:
This resulted in typescript inferring the type of `TestPluginRegistry` to `never` since `load`, `loadModuleFromUrl` methods are public in this declaration, but private in the actual `PluginRegistry` definition. So removed this custom definition and added `@ts-ignore` in call sites to get unblocked.